experimental/SetPartitions/src/Util.jl

```diff
@@ -0,0 +1,95 @@
+"""
+    helper_new_point_values_vector(p::Vector{Vector{Int}}, q::Vector{Vector{Int}})
+
+Return a semantically identical partition in form of an vector, 
+which has new number values.
+"""
+function helper_new_point_values_vector(p::Vector, q::Vector)
```

Comment:
Function decl does not match the docstring, why?
```suggestion
function helper_new_point_values_vector (p::Vector{Vector{Int}}, q::Vector{Vector{Int}})
```